src/shared/src/types/change-request-types.ts

```diff
@@ -8,7 +8,10 @@ import { WbsNumber } from './project-types';
 
 export interface ChangeRequest {
   crId: number;
-  wbsNum: WbsNumber;
+  wbs: {
```

Comment:
i think the nesting is a little overcomplicated and hard to read

instead we should have just `wbsNum` and `wbsName` as properties